pkg/katautils/config.go

```diff
@@ -994,6 +993,19 @@ func SetKernelParams(runtimeConfig *oci.RuntimeConfig) error {
 		}
 	}
 
+	// set the scsi scan mode to async for virtio-scsi
+	if runtimeConfig.HypervisorConfig.BlockDeviceDriver == config.VirtioSCSI {
+		if !strings.Contains(formatted, "scsi_mod.scan") {
```

Comment:
This looks odd - shouldn't the value be tested? What if `scsi_mod.scan=none` or `scsi_mod.scan=sync`?

- https://github.com/torvalds/linux/blob/master/Documentation/scsi/scsi-parameters.txt#L95..L98

Reply:
also, kata-agent already scans the SCSI busses [1], `scsi_mod.scan=none` should work, right?

[1] https://github.com/kata-containers/agent/blob/649d44117a862cdcc72dfef6f3ade3bdec6e3fc4/device.go#L322

Reply:
Yes, I think we can get away with passing  `scsi_mod.scan=none`, since all scsi devices are hotplugged in our case and the agent scans the scsi bus after the scsi device is hotplugged.
@justin-he Give that a shot and see how it goes.

Reply:
> This looks odd - shouldn't the value be tested? What if `scsi_mod.scan=none` or `scsi_mod.scan=sync`?
> 
> * https://github.com/torvalds/linux/blob/master/Documentation/scsi/scsi-parameters.txt#L95..L98

My original purpose is:
1. If the user specify any parameter for scsi_mod.scan, using that specified one without checking.
2. If the user doesn't specify, using "scsi_mod.scan=async"
![image](https://user-images.githubusercontent.com/44389508/77613598-0c2f5f80-6f66-11ea-9190-bd8d013b47e3.png)

Reply:
> Yes, I think we can get away with passing `scsi_mod.scan=none`, since all scsi devices are hotplugged in our case and the agent scans the scsi bus after the scsi device is hotplugged.
> @justin-he Give that a shot and see how it goes.

Ok, I will let you know the result once I test it. suspect that "none" will be better than "async"

Reply:
scsi_mod.scan=sync
[    0.113828] [    T1] scsi host0: Virtio SCSI HBA
[    0.134006] [    T1] tun: Universal TUN/TAP device driver, 1.6
scsi_mod.scan=none
[    0.105891] [    T1] scsi host0: Virtio SCSI HBA
[    0.107868] [    T1] tun: Universal TUN/TAP device driver, 1.6